src/lxc/lxccontainer.h

```diff
@@ -1017,6 +1017,22 @@ int list_active_containers(const char *lxcpath, char ***names, struct lxc_contai
  */
 int list_all_containers(const char *lxcpath, char ***names, struct lxc_container ***cret);
 
+struct lxc_log_conf {
+	const char *name;
+	char *file;
+	char *priority;
+	const char *prefix;
+	int quiet;
+	const char *lxcpath;
```

Comment:
Sot the struct should be:

    struct lxc_log {
            const char *name;
            const char *lxcpath;
            char *logfile;
            char *loglevel;
            char *logprefix;
            bool quiet;
    };

Why aren't the other not marked as `const`? Any reason for that?